save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/benchmarks/BenchmarkEntity.kt

```diff
@@ -19,6 +20,18 @@ enum class BenchmarkCategoryEnum {
     PERFORMANCE,
     STATIC_ANALYSIS,
     ;
+
+    companion object : TabMenuBar<BenchmarkCategoryEnum> {
+        // The string is the postfix of a [regexForUrlClassification] for parsing the url
+        private val postfixInRegex = values().map { it.name.lowercase() }.joinToString { "|" }
+        override val defaultTab = ALL
+        override val regexForUrlClassification: Regex = Regex("/project/[^/]+/[^/]+/($postfixInRegex)")
+        override var paths: Pair<String, String> = "" to ""
```

Comment:
Why are both empty?

Reply:
I set paths in Views because I can't set their values in advance, they depend on the name of projects, organizations, etc.